src/com/palmergames/bukkit/towny/event/DeleteNationEvent.java

```diff
@@ -51,10 +56,27 @@ public long getNationCreated() {
 	}
 	
 	/**
+     * @deprecated in favor of {@link #getKingUUID()} 
 	 * @return deleted nation king uuid.
 	 */
 	@Nullable
 	public UUID getNationKing() {

```

Comment:
```suggestion
	/**
     * @deprecated since 0.98.4.2 in favor of {@link #getLeaderUUID()} 
	 * @return deleted nation leader uuid.
	 */
	@Nullable
	@Deprecated
	public UUID getNationKing() {

```